build.gradle

```diff
@@ -51,6 +52,7 @@ dependencies {
 
     // Spring
     implementation group: 'org.springframework.retry', name: 'spring-retry'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-autoconfigure'
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jdbc'
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
```

Comment:
`spring-boot-autoconfigure` moved up from line 108, where it was hidden in with otel dependencies